Core/include/Acts/Propagator/StandardAborters.hpp

```diff
@@ -161,23 +167,31 @@ struct SurfaceReached {
 
       // reaching the target calls a navigation break
       navigator.targetReached(state.navigation, true);
-    } else {
-      // Target is not reached, update the step size
-      const double overstepLimit = stepper.overstepLimit(state.stepping);
-      // Check the alternative solution
-      if (distance < overstepLimit and sIntersection.alternative) {
-        // Update the distance to the alternative solution
-        distance = sIntersection.alternative.pathLength;
-      }
-      stepper.setStepSize(state.stepping, distance, ConstrainedStep::aborter,
-                          false);
 
-      ACTS_VERBOSE("Target: 0 | "
-                   << "Target stepSize (surface) updated to "
-                   << stepper.outputStepSize(state.stepping));
+      return true;
     }
-    // path limit check
-    return targetReached;
+
+    const double pLimit =
+        state.stepping.stepSize.value(ConstrainedStep::aborter);
+    // not using the stepper overstep limit here because it does not always work
+    // for perigee surfaces
+    const double oLimit = overstepLimit;
```

Comment:
I needed to increase the overstep limit here because perigee surface do not play as nice as plane surfaces do

before my changes we ignored the overstepping tolerance completely but I don't think that is great either